### PR TITLE
Dead Pull - Romcenter.Romcenter version 4.1.1.51

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Romcenter.Romcenter
+PackageVersion: 4.1.1.51
+PackageLocale: en-US
+Publisher: Romcenter
+PackageName: Romcenter
+License: Copyright © Eric Bole-Feysot
+ShortDescription: Romcenter Setup
```
Comment (ItzLevvie) [inline manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.locale.en-US.yaml:10]:
```
ShortDescription: Romcenter Setup
```
Please write a better description for what this application is about.

File: manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Romcenter.Romcenter
+PackageVersion: 4.1.1.51
```
Comment (ItzLevvie) [inline manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1.1
```

File: manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Romcenter.Romcenter
+PackageVersion: 4.1.1.51
```
Comment (ItzLevvie) [inline manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.yaml:5]:
```suggestion
PackageVersion: 4.1.1
```

File: manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Romcenter.Romcenter
+PackageVersion: 4.1.1.51
```
Comment (ItzLevvie) [inline manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.1
```

File: manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Romcenter.Romcenter
+PackageVersion: 4.1.1.51
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.romcenter.com/downloadfile.php?file=romcenter32_4.1.1.exe
+  InstallerSha256: 3A27EF7BCF7C6C697C43C3C468A3502AFBC53860187397D43F1A4980FF2B6B4E
```
Comment (ItzLevvie) [inline manifests/r/Romcenter/Romcenter/4.1.1.51/Romcenter.Romcenter.installer.yaml:10]:
```suggestion
  InstallerSha256: 3A27EF7BCF7C6C697C43C3C468A3502AFBC53860187397D43F1A4980FF2B6B4E
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Masamune3210** [comment]
How do I re-path the file in the commit to satisfy the new version name?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You can rename it using GitHub Desktop from https://desktop.github.com - make sure to checkout `autogenerated/Romcenter.Romcenter/6a14c823-b95c-449c-8071-883c125ec8de`.

**Comment by Masamune3210** [comment]
Unless you want to help me more I'm just going to make a new pull request with the corrected files since I seem to have royally messed this one up with my lack of git-fu. I still cant find out how to successfully move the files in the filetree either

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> Unless you want to help me more I'm just going to make a new pull request with the corrected files since I seem to have royally messed this one up with my lack of git-fu. I still cant find out how to successfully move the files in the filetree either

For the web version take a look at https://docs.github.com/en/github/managing-files-in-a-repository/managing-files-on-github/renaming-a-file

For the client version:
1) Fork your winget-pkgs repository
2) Switch your branch to `Masamune3210:autogenerated/Romcenter.Romcenter/6a14c823-b95c-449c-8071-883c125ec8de`
3) Open up File Explorer, and go to `manifests\r\Romcenter.Romcenter` and change `4.1.1.51` to `4.1.1`
4) Bring up the client again and commit the changes to your branch by typing in a summary, etc. and push the changes.

**Pull request closed without merging**